python/src/energuide/extractor.py

```diff
@@ -46,6 +48,16 @@ def _read_csv(filepath: str) -> typing.Iterator[reader.InputData]:
 def _extract_snippets(data: typing.Iterable[reader.InputData]) -> typing.Iterator[reader.InputData]:
     for row in data:
         row['ForwardSortationArea'] = row['CLIENTPCODE'][:3]
+
+        doc = ElementTree.fromstring(row['RAW_XML'])
+        house = doc.find('House')
+        if house:
+            extra_data = snippets.snip_house(house) if house else None
```

Comment:
`if house else None` seems unnecessary here since in the line directly above it `house` is being tested in the if statement

Reply:
good catch